src/components/RadioGroup/RadioGroup.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { errorText, helpText } from '../../stories/helpers';
+
+import RadioGroup from './RadioGroup';
+import InputRadio from '../InputRadio';
+
+const meta = {
+  title: 'Components/Radio group',
+  component: RadioGroup,
+  args: {
+    legend: 'Legend'
+  }
+} satisfies Meta<typeof RadioGroup>;
+
+export default meta;
+type Story = StoryObj<typeof RadioGroup>;
+
+export const Playground: Story = {
+  render: (args) => (
+    <RadioGroup {...args}>
+      <InputRadio label="Radio 1" name="groupDefault" value="defaultRadio1" defaultChecked />
+      <InputRadio label="Radio 2" name="groupDefault" value="defaultRadio2" />
+      <InputRadio label="Radio 3" name="groupDefault" value="defaultRadio3" />
+    </RadioGroup>
+  )
+};
+
+export const WithHelp: Story = {
+  name: 'With help',
+  args: {
+    helpText
+  },
+  render: (args) => (
+    <RadioGroup {...args}>
+      <InputRadio label="Radio 1" name="groupDefault" value="defaultRadio1" defaultChecked />
+      <InputRadio label="Radio 2" name="groupDefault" value="defaultRadio2" />
+      <InputRadio label="Radio 3" name="groupDefault" value="defaultRadio3" />
```

Comment:
the name and value attr values are repeated across examples. can we make them unique for the docs pg